tests/e2e/tests/test_storage_fsx.py

```diff
@@ -59,7 +59,9 @@ def kustomize_path():
 
 class TestFSx:
     @pytest.fixture(scope="class")
-    def setup(self, metadata, kustomize, patch_kfp_to_disable_cache, port_forward, static_provisioning):
+    # todo: revert once cache deployer fix is merged
```

Comment:
what does this has to do with cache deployer ?

Reply:
the line below has the setup without the `patch_kfp_to_disable_cache` so we don't get a failure from applying patches to the cache deployer deployment pod